src/app/user-settings/user-settings.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, OnInit } from '@angular/core';
+import {User} from '../shared/domain/user.domain';
+import {MusicFolder} from '../shared/domain/music-directory.domain';
+import {MusicDirectoryService} from '../shared/service/music-directory.service';
+import {NotificationService} from '../shared/service/notification.service';
+import {UserService} from '../shared/service/user.service';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-user-settings',
+  templateUrl: './user-settings.component.html',
+  styleUrls: ['./user-settings.component.scss']
+})
+export class UserSettingsComponent implements OnInit {
+  users: Array<User> = [];
+  user: User = {} as User;
+  selUser: User = {} as User;
+  folders: Array<MusicFolder> = [];
+  newUserName: string;
+  newPassword: string;
+  newPasswordConf: string;
+
+  bitRates = [
+    {rate: '', text: 'No Limit'},
+    {rate: 32, text: '32 Kbps'},
+    {rate: 40, text: '40 Kbps'},
+    {rate: 48, text: '48 Kbps'},
+    {rate: 56, text: '56 Kbps'},
+    {rate: 64, text: '64 Kbps'},
+    {rate: 80, text: '80 Kbps'},
+    {rate: 96, text: '96 Kbps'},
+    {rate: 112, text: '112 Kbps'},
+    {rate: 128, text: '128 Kbps'},
+    {rate: 160, text: '160 Kbps'},
+    {rate: 192, text: '192 Kbps'},
+    {rate: 224, text: '224 Kbps'},
+    {rate: 256, text: '256 Kbps'},
+    {rate: 320, text: '320 Kbps'},
+  ];
+
+  constructor(private directoryService: MusicDirectoryService, private userService: UserService,
+              private notificationService: NotificationService, private router: Router) { }
+
+  ngOnInit() {
+    this.refreshUserList();
+
+    this.directoryService.getMusicFolders().subscribe(
+      data => this.folders.push(...data),
+      err => this.notificationService.notify(err)
```

Comment:
Yup notification service is not yet up. Gonna do this once I cleaned my current work

Reply:
primeng have a nice message service and very straight forward to set up.  I already have code that wraps their service and allows the user to push notifications using this._alertservice.error("Title", "Message"); this._alertservice.info("Title", "Message"); etc. If you want i can hook it into this and create a pull request for it.

Reply:
@atecey Hey gonna take a look at PrimeNG, seems more like a complete solution that just a module. Let's see if we want to ship the entire thing in. I was more checking for modules like https://github.com/scttcper/ngx-toastr

Reply:
The nice thing about PrimeNG is you can just load in what you need.  It has some nice responsive components all they need is styling.  Plus it adds next to nothing to the final build size.  That's a nice toast module.